modules/002-deckhouse/openapi/config-values.yaml

```diff
@@ -161,6 +164,31 @@ properties:
               The update mechanism ensures that Deckhouse will not be updated before the specified time.
 
               When using update windows, the Deckhouse update after the notification will happen at the nearest possible update window, but not before the time specified in `minimalNotificationTime` expires.
+          auth:
+            type: object
```

Comment:
1. There should be validation that either basic auth or token can be provided.
2. The naming can be the same as for PrometheusRemoteWrite (I think we use the same option names in Upmeter too).

https://deckhouse.io/documentation/v1/modules/300-prometheus/cr.html#prometheusremotewrite-v1-spec
```
bearerToken:
basicAuth:
  username:
  password:
```
And with this structure, it is easier to validate that only one field is provided.

3. We usually use the `auth` key to describe module authentication params (that will be used to authenticate and authorize users to get access to this module). Still, maybe it is ok because, in this case, `auth` is a nested field.

Reply:
Improved.